astroid/nodes/node_classes.py

```diff
@@ -2016,13 +2020,17 @@ def postinit(self, nodes: typing.List[NodeNG]) -> None:
         """
         self.nodes = nodes
 
-    def scope(self):
+    def scope(self) -> "LocalsDictNodeNG":
         """The first parent node defining a new scope.
 
         :returns: The first parent scope node.
         :rtype: Module or FunctionDef or ClassDef or Lambda or GenExpr
```

Comment:
```suggestion
        :rtype: LocalsDictNodeNG
```

Reply:
I've never really checked but I guess sphinx should be able to create the rtype from the typing information in the signature ?

Reply:
I have no idea. I never really used `sphinx`. We might want to test this at some point and remove some of the redundant doc strings.